.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -1,31 +1,151 @@
 name: "\U0001F41E Bug report"
-description: Report an issue
-labels: ['s: pending triage'] # This will automatically assign the 's: pending triage' label
+description: "Something isn't working as expected \U0001F914"
+labels: ['s: pending triage', 'c: bug']
 body:
   - type: markdown
     attributes:
-      value: Thanks for taking the time to fill out this bug report!
+      value: |
+        ## :warning: We use GitHub Issues to track bug reports, feature requests and regressions
+
+        If you are not sure that your issue is a bug, you could:
+
+        - use our [Discord community](https://chat.fakerjs.dev)
+        - open a new [discussion](../discussions) and ask your question there
+
+        **NOTE:** You don't need to answer questions that you know that aren't relevant.
+
+  - type: checkboxes
+    attributes:
+      label: 'Is there an existing issue for this?'
+      description: 'Please search [here](./?q=is%3Aissue) to see if an issue already exists for the bug you encountered'
+      options:
+        - label: 'I have searched the existing issues'
+          required: true
+
+  - type: checkboxes
+    attributes:
+      label: 'Have you checked the documentation?'
+      description: 'Please read our [documentation](https://fakerjs.dev/guide) to see if the bug you encountered is a use error'
```

```suggestion
      description: 'Please read our [documentation](https://fakerjs.dev/guide) to see if the bug you encountered is a use error'
```
Did you mean to say "user error"?

---

I'm sorry I don't know the English term. I was trying to describe an error that occurs because you are the one using the code incorrectly. Is that what a "user error" is? Google was suggesting "use error" for me^^

---

> I'm sorry I don't know the English term. I was trying to describe an error that occurs because you are the one using the code incorrectly. Is that what a "user error" is? Google was suggesting "use error" for me^^

No worries! It's "user error", I don't know why Google is saying "use error" 😅